localstack-core/localstack/packages/java.py

```diff
@@ -56,6 +56,7 @@ def get_java_env_vars(self, path: str = None) -> dict[str, str]:
 
         return {
             "JAVA_HOME": java_home,
+            "LD_LIBRARY_PATH": f"{java_home}/lib:{java_home}/lib/server",
```

Comment:
This would effectively overwrite an existing `LD_LIBRARY_PATH` wherever we use the Java mixin / this function. I think this should be handled the same way as for the path.

Reply:
@alexrashed but shouldn't this be treated more like `JAVA_HOME` and localised to the Java environment to be used? An invalid exisiting `LD_LIBRARY_PATH` value might potentially poison the environment.

Reply:
Sure, there definitely shouldn't be a faulty `LD_LIBRARY_PATH`, but couldn't it be necessary to have the shared libraries of Java as well as other libraries in the `LD_LIBRARY_PATH`?
For example in OpenSeach we add custom libraries for KNN to the `LD_LIBRARY_PATH` here:
https://github.com/localstack/localstack/blob/de272085dbdbbd594f797805e71ac2b2fd126749/localstack-core/localstack/services/opensearch/cluster.py#L479

I know that for opensearch we use the Java runtime from the package, but that would be an example where we would have to have multiple paths on the `LD_LIBRARY_PATH`.